src/main/java/taxi/service/DriverService.java

```diff
@@ -3,4 +3,5 @@
 import taxi.model.Driver;
 
 public interface DriverService extends GenericService<Driver> {
+    Driver findByLogin(String login);
```

Comment:
Task: "Create new method Optional<Driver> findByLogin(String login) in DriverDao and DriverService." We shoult return Optional<Driver>  from service too